examples/features/health/client/main.go

```diff
@@ -20,26 +39,30 @@ var serviceConfig = `{
 	}
 }`
 
-const (
-	exampleScheme      = "example"
-	exampleServiceName = "health.example.grpc.io"
-)
-
-var addrs = []string{"localhost:50051", "localhost:50052"}
-
 func callUnaryEcho(c pb.EchoClient) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
 	r, err := c.UnaryEcho(ctx, &pb.EchoRequest{})
-	if err == nil {
+	if err != nil {
+		fmt.Println(fmt.Sprintf("UnaryEcho: _, %v", err))
```

Comment:
This can be simplified a bit:

`fmt.Printf("UnaryEcho: _, %v\n", err)`
or
`fmt.Println("UnaryEcho: _, ", err)`